README.md

```diff
@@ -18,6 +18,19 @@ Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in
 
 ---
 
+### 16 September 2024
+#### Self Assessment BISS API
+
+The following change is now available in Sandbox and Production for v2.0.
+
+##### Changed
+
+The following endpoint no longer accepts data for tax years 2025-26 onwards:
+
+- Retrieve a Business Income Source Summary
+
+---
```

Comment:
Please change the language of the "Changed" section as follows to maintain consistency with other similar updates:-

Retrieve a Business Income Source Summary:
- Error RULE_TAX_YEAR_NOT_SUPPORTED is returned for requests submitted for tax years 2025-26 onwards.

Reply:
done